src/app/components/favorites/favorites.component.html

```diff
@@ -1,14 +1,13 @@
-<form id="form" (ngSubmit)="addFavorite()">
-  <ion-header>
-  <ion-toolbar class="toolbar">
-
+<form id="favoriteForm" (ngSubmit)="addFavorite()">
+  <ion-header>  
+    <ion-toolbar class="toolbar">
       <ion-item  class="inputItem">
           <ion-input [(ngModel)]="favorite"  name='favoriteLocation' placeholder="Location"></ion-input>
       </ion-item>
     </ion-toolbar> 
 
-      <ion-button id="addFavoriteBtn" collapse="true"  type="submit" expand="full" (click)="addFavorite()">Add Favorite</ion-button>
-    </ion-header>
+    <ion-button id="addFavoriteBtn" collapse="true"  type="submit" expand="full" (click)="addFavorite()">Add Favorite</ion-button>
```

Comment:
maybe instead of writing Add Favorite, UI team can add a "heart" or "star" button.

Reply:
I think a star would look better.